### PR TITLE
Remove diagnosis when peer returns

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: src/device/pf_pdport.c [hunk 1]
```diff
@@ -752,6 +787,7 @@ static void pf_pdport_run_peer_check (pnet_t * net, int loc_port_num)
 
    if (p_port_data->pdport.check.active == true)
    {
+      pf_pdport_remove_missing_peer_diagnosis (net, loc_port_num);
```
Comment (olbjo) [inline src/device/pf_pdport.c:790]: Maybe better name is "pf_pdport_check_missing_peer_diagnosis" ?

Comment (olbjo) [reply src/device/pf_pdport.c:790]: Or even better:
we remove "missing peer" in logs/naming/comments, 
The alarm is named "no peer detected" and we should use this instead.

So, new  suggested name is : "pf_pdport_check_no_peer_detected_diagnosis "

also replace use of "missing peer" with "no peer detected"

**Review by olbjo — CHANGES_REQUESTED** [review]
Great!
Name changes suggested. Lets discuss.

File: src/device/pf_pdport.c (hunk 1 above)
Comment (pyhys) [reply src/device/pf_pdport.c]: Done

**Review by olbjo — APPROVED** [review]
Great!

**Pull request merged**